app/routes/events/view/edit/attendee.js

```diff
@@ -0,0 +1,47 @@
+import Route from '@ember/routing/route';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+import { A } from '@ember/array';
+export default Route.extend(EventWizardMixin, {
+
+  titleToken() {
+    return this.l10n.t('Attendee Form');
+  },
+
+  async model() {
+    let filterOptions = [{
+      name : 'form',
+      op   : 'eq',
+      val  : 'attendee'
+    }];
+
+    let data = {
+      event: this.modelFor('events.view')
+    };
+    data.customForms = await data.event.query('customForms', {
+      filter       : filterOptions,
+      sort         : 'id',
+      'page[size]' : 50
```

Comment:
Is the page[size] same for other forms as well?

Reply:
@mrsaicharan1 Yes